### PR TITLE
Handle #rage and #please-im-lost

### PR DESCRIPTION
**Pull request by antoine-royer**
Adding commands for rage and please im lost.

### CODE REVIEW

File: cogs/fun.py [hunk 1]
```diff
@@ -3,5 +3,15 @@
...
 class Fun(commands.Cog):
-    def __init__(self, bot):
+    
+    def __init__(self, bot, config):
         self.bot = bot
+        self.config = config
+
+    @commands.Cog.listener()
+    async def on_message(self, message):
+        if message.channel.id == self.config["ID"]["RAGE"] and not message.content.isupper():
```
Comment (quentinguidee) [inline cogs/fun.py:13]: Does this accepts emojis ?

Comment (quentinguidee) [reply cogs/fun.py:13]: And images

**Comment by Guiguim255** [comment]
Deprecate since #16, we should close this draft.

**Comment by Guiguim255** [comment]
@Tim-ats-d @quentinguidee

**Pull request closed without merging**